realm/realm-library/src/main/cpp/jni_util/jni_utils.hpp

```diff
@@ -31,12 +35,16 @@ class JniUtils {
 
     // Call this only once in JNI_OnLoad.
     static void initialize(JavaVM* vm, jint vm_version) noexcept;
+    // Call this in JNI_OnUnload.
+    static void release();
```

Comment:
`release()` is corresponding to the above `initialize()`. So it is supposed to do more clean up work if needed. (But actually there is no practical way to unload a so, so the `JNI_OnUnload()` will never be called 😁  )